src/main/java/frc/robot/commands/finishScore.java

```diff
@@ -10,6 +10,11 @@
 
 public class finishScore extends SequentialCommandGroup {
   public finishScore(Lift lift) {
+    /** if scoring action is cancelled, then don't run finish score and reset cancelScore */
+    if (lift.getCancelScore()) {
+      lift.setCancelScore(false);
+      return;
```

Comment:
While this does work, it is probably better practice to just call `this.cancel()` (documentation [here](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Command.html#cancel())).

Reply:
Using `isFinished()` and a boolean is also an option, but is not as intended to be used for *cancelling* which is what we're doing here. Functionally for our purposes there is no difference, but it's good for the code to be clear with as little extra information as possible.